### DISCUSSION TITLE
Desktop link for Crypto.com is broken

### DISCUSSION THREAD

**Discussion by Adamj1232** [WalletConnect/walletconnect-monorepo#1422] [Developer Support]
**Describe the bug**
When clicking the Crypto.com option within the Desktop tab is broken and opens a tab with a broken link.

**To Reproduce**
1. Go to Curve.fi
2. Click on Connect wallet
3. Select Desktop tab
4. Click crypto.com and see broken link

**Expected behavior**
Takes a user to the proper location

**Desktop (please complete the following information):**
 - OS: Mac
 - Browser Chrome, Firefox, Brave

**Comment by looter** [comment]
@Adamj1232 thanks for flagging this. question for you do you have the crypto.com desktop wallet installed? This should be a deeplink and if you have the desktop wallet installed it should route it accordingly.

**Comment by Adamj1232** [comment]
I do not but feel there should be some user feedback if that is the case since it is the first option displayed by default

**Comment by JesseKuntz** [comment]
@looter I'm having some issues with related to WC v2 and these so-called deeplinks, and bringing it up here because this is the first place that I'm seeing it mentioned - if there's a better place to pose this question, please let me know 😄 

I totally get how deeplinks are supposed to function, but I'm feeling like there's an edge case: what if there's no desktop app for a specific wallet, just a web app?

What has been happening to me is I'll send a `eth_sendTransaction` request, and because the wallet in question (Fireblocks) is set in `WALLETCONNECT_DEEPLINK_CHOICE`, it'll open the wallet in the current tab, thus closing the dApp. We'd like to keep the dApp open, and open the wallet in a new tab instead of the current tab if the `WALLETCONNECT_DEEPLINK_CHOICE` isn't actually going to take us to a desktop app. Would you happen to know of a way that this is possible?

**Comment by looter** [comment]
@JesseKuntz Thanks for raising with additional details! I've shared with the Engineering team.
